docs/source/admin/traffic_ops/using.rst

```diff
@@ -293,135 +293,136 @@ The fields in the Delivery Service view are:
+| Global Max Mbps                       | The maximum bits per second this delivery service can serve across all EDGE caches before traffic will be diverted to the bypass destination. For a DNS delivery service, the Bypass Ipv4 or Ipv6  will be used   |
+|                                       | (depending on whether this was a A or AAAA request), and for HTTP delivery services the Bypass FQDN will be used.                                                                                                 |
++---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Global Max TPS                        | The maximum transactions this delivery service can serve across all EDGE caches before traffic will be diverted to the bypass destination. For a DNS delivery service, the Bypass Ipv4 or Ipv6  will be used      |
+|                                       | (depending on whether this was a A or AAAA request), and for HTTP delivery services the Bypass FQDN will be used.                                                                                                 |
++---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| HTTP Bypass FQDN                      | (for HTTP routed delivery services only) This is the FQDN Traffic Router will redirect to (with the same path) when the max Bps or Max Tps for this deliveryservice are exceeded.                                 |
++---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Info URL                              | Info URL  for this delivery service. To be consumed from the APIs by downstream tools (Portal).                                                                                                                   |
++---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Initial Dispersion                    | For HTTP delivery services - the number of caches that will be used by Traffic Router.                                                                                                                            |
```

Comment:
Suggest to add something like "If caches reach capacity, additional caches will be added as needed"

Reply:
done